static/styles/left-sidebar.scss

```diff
@@ -337,7 +337,7 @@ li.top_left_starred_messages {
     which also affects it positioning.
 */
 .topic-sidebar-arrow {
-    top: 1px;
+    top: 25px;
```

Comment:
It's very unlikely 25px is correct here; did you test this change?

Reply:
It is a typo mistake. I write 2.5 when I tested it and because I was looking something else in the file I closed the text editor afterwords and accidentally that happened. I am very sorry.